docs/products/kafka/kafka-mirrormaker/concepts/replication-in-mirrormaker2.rst

```diff
@@ -0,0 +1,60 @@
+Replication in Apache Kafka® MirrorMaker 2
```

Comment:
Since it's all under MirrorMaker 2, probably makes sense to make this as `Replication FAQ`